app/components/candidate_interface/offer_review_component.rb

```diff
@@ -5,12 +5,13 @@ def initialize(course_choice:)
     end
 
     def rows
-      [
+      rows = [
         provider_row,
         course_row,
         location_row,
-        conditions_row,
       ]
+      rows << conditions_row if @course_choice.offer&.dig('conditions')&.present?
```

Comment:
this is a bit clunky but i'm not sure if this is going to return nil or an empty array so it guards against both